File: KYOSAIWEB/Models/S0050.cs
namespace KYOSAI_WEB.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;
    using System.Web.Mvc;

    [Table("TELAS.S0050")]
    public partial class S0050
    {
        [Key]
        [Required(ErrorMessage = "必須項目")]
        [AllowHtml]
        [StringLength(20, ErrorMessage = "ユーザーIDは、は20文字以下でなければなりません。")]
        public string USERID { get; set; }

        [StringLength(32)]
        public string USERNM { get; set; }

        [AllowHtml]
        [Required(ErrorMessage = "必須項目")]
        [DataType(DataType.Password)]
        [StringLength(96, ErrorMessage = "パスワードは、は20文字以下でなければなりません。")]
        public string PASS { get; set; }

        public DateTime? PSWUPDTDT { get; set; }

        [StringLength(1)]
        public string LOGINHIST { get; set; }

        [StringLength(6)]
        public string SUPPCD { get; set; }

        [StringLength(4)]
        public string CHARGECD { get; set; }
    }
}

File: YMESWEB/Models/s0050.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.Data.Entity.Spatial

<Table("telas.s0050")>
Partial Public Class s0050
    Public Sub New()
        s0060 = New HashSet(Of s0060)()
    End Sub

    <Key>
    <StringLength(20)>
    Public Property userid As String

    <Required>
    <StringLength(32)>
    Public Property usernm As String

    <StringLength(96)>
    Public Property pass As String

    <Required>
    <StringLength(64)>
    Public Property instid As String

    Public Property instdt As Date

    <Required>
    <StringLength(50)>
    Public Property instterm As String

    <Required>
    <StringLength(50)>
    Public Property instprgnm As String

    <Required>
    <StringLength(64)>
    Public Property updtid As String

    Public Property updtdt As Date

    <Required>
    <StringLength(50)>
    Public Property updtterm As String

    <Required>
    <StringLength(50)>
    Public Property updtprgnm As String

    Public Property pswupdtdt As Date?

    <StringLength(1)>
    Public Property loginhist As String

    Public Overridable Property s0060 As ICollection(Of s0060)
End Class
